general/src/protocol/event.rs

```diff
@@ -172,6 +172,8 @@ pub struct GroupingConfig {
     /// The id of the grouping config.
     #[metastructure(max_chars = "enumlike")]
     pub id: Annotated<String>,
+    /// The enhancements configuration.
+    pub enhancements: Annotated<String>,
```

Comment:
What are enhancements, again?

Reply:
It's a base64 gzip encoded msgpack dump of the parsed rules from the ruleset that defines how frames should be processed.